README.md

```diff
@@ -2,8 +2,23 @@
 
 pg_duckdb is a Postgres extension that embeds DuckDB's columnar-vectorized analytics engine and features into Postgres. We recommend using pg_duckdb to build high performance analytics and data-intensive applications.
 
+Third party developers will be able to extend both DuckDB and the PostgreSQL integration by writing their own DuckDB extensions. Those extensions will be distributed through the DuckDB community extension repository.
+
 pg_duckdb was developed in collaboration with our partners, [Hydra](https://hydra.so) and [MotherDuck](https://motherduck.com).
 
+
+## Roadmap
+
+* Create and manage analytical tables with good performance on analytical queries
+* Broad support for Postgres types and functions within analytical context
+* Query external datasets on S3, GCP and Azure (Parquet, Iceberg and Delta Lake)
+* Join native Postgres and analytical data efficiently
+* Seamlessly run queries read and write queries against local data and MotherDuck tables
+* Allow for easy configuration and deployment of DuckDB extensions
+* Columnar indexing of native Postgres tables
```

Comment:
I don't quite understand this one, I'd cut it.

Reply:
Let me get clarification Heikki before I cut this (this section is a combination of his roadmap (based on a list from Boaz) and our Goals section.

Reply:
Going to strike this and add back if/when I get clarification on functionality.